packages/hub/utils/scheduled-payments.ts

```diff
@@ -3,7 +3,8 @@ import { convertDateToUTC } from './dates';
 // This function will return the next payment date based on the frequency.
 // fromDate will be either the current date when a recurring scheduled is created,
 // or the date of the last payment.
-export function calculateNextPayAt(fromDate: Date, recurringDay: number) {
+// When a recurring scheduled is created the recurringUntil parameter must be not undefined.
```

Comment:
If you add the default value for `recurringUntil` then this comment won't be needed